pkg/controller/cassandracluster/cassandra_status_test.go

```diff
@@ -216,9 +226,12 @@ func helperCreateCassandraCluster(t *testing.T, cassandraClusterFileName string)
 	assert.Equal(cc.Status.Phase, api.ClusterPhaseRunning)
 
 	for _, dcRackName := range dcRackNames {
```

Comment:
You don't really need a variable, just delete `dcRackNames` and use `cc.GetDCRackNames()` here.